app/src/main/res/navigation/nav_transactions.xml

```diff
@@ -6,12 +6,12 @@
     app:startDestination="@id/transaction_lists_fragment">
     <fragment
         android:id="@+id/transactions_details_fragment"
-        android:name="com.kwartracker.android.transactions.ui.details.TransactionDetailsFragment"
+        android:name="com.kwartracker.android.transactions.ui.details.DetailsTransactionFragment"
         android:label="Transaction Details"
         tools:layout="@layout/fragment_transaction_details" />
     <fragment
         android:id="@+id/transaction_lists_fragment"
-        android:name="com.kwartracker.android.transactions.ui.list.TransactionsListFragment"
+        android:name="com.kwartracker.android.transactions.ui.list.ListTransactionsFragment"
```

Comment:
@danacealfonso 
```suggestion
        android:name="com.kwartracker.android.transactions.ui.list.TransactionListFragment"
```
Provide more comprehensible class name. It is best for class names in singular form. If it is a collection/list, name it to singular form + list.

Reply:
fix in 8e6fa36a048fe726e08f61764cfae1f06a381b94